contributors/design-proposals/storage/data-source.md

```diff
@@ -46,18 +48,16 @@ type PersistentVolumeStatus struct {
 
 type PersistentVolumeConditionType string
 
-// These are valid conditions of Pvc
+// These are valid conditions of PersistentVolume
 const (
 	// An user trigger resize of pvc has been started
```

Comment:
PersistentVolume rather than pvc?

Reply:
Yes, this condition describes whether data is populated to a PersistentVolume.

The comments should be fixed though.  This is not about "resize".